src/stravalib/protocol.py

```diff
@@ -314,7 +319,7 @@ def _request(
         raw = requester(url, params=params)  # type: ignore[operator]
         # Rate limits are taken from HTTP response headers
         # https://developers.strava.com/docs/rate-limits/
-        self.rate_limiter(raw.headers)
+        self.rate_limiter(raw.headers, method)
```

Comment:
Passing the method to the rate limiter will allow us to choose wether to use the overall rate limits or the specific read rate limits (for GET requests)